third_party/openthread/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+import("//build_overrides/chip.gni")
+import("//build_overrides/openthread.gni")
+
+import("${chip_root}/src/platform/device.gni")
```

Comment:
Unneeded import?

Reply:
it is needed for `chip_openthread_ftd` argument declared in that gni.